pkgs/development/libraries/redkite/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitLab, cmake, cairo }:
+
+stdenv.mkDerivation rec {
+  pname = "redkite";
+  version = "1.0.0";
+
+  src = fetchFromGitLab {
+    owner = "iurie-sw";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1qg52m6pqilyq601qlwyp7dj921gis2j3vg7x5dpwrkb85iml71n";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ cairo ];
+
+  enableParallelBuilding = true;
+
+  meta = {
+    homepage = "https://gitlab.com/iurie-sw/redkite";
+    description = "A small GUI toolkit";
+    license = stdenv.lib.licenses.gpl3;
```

Comment:
Readme says GPL 3 but [source code](https://gitlab.com/iurie-sw/redkite/-/blob/7e15627dfd54c748129fe1cff9a4f83f034b1d9a/src/RkContainer.cpp#L9-11) says +. Needs clarification from upstream.

Reply:
Good catch!
Do you always read the source of packages you review?

https://github.com/iurie-sw/redkite/issues/25

Reply:
Only sometimes, mostly when checking for license or when something looks fishy.

Reply:
All done.

I also changed the geonkick license to gpl3+, since that is what the code says.
I asked upstream to unify that too.

Reply:
https://github.com/iurie-sw/redkite/issues/25#issuecomment-643269349